src/components/energiatodistus-haku/schema.js

```diff
@@ -51,6 +51,12 @@ const contains = {
   format: R.curry((command, key, value) => [[command, key, `%${value}%`]])
 };
 
+const contains_no = {
```

Comment:
Tämä pitäis varmaan olla camelCase?

Reply:
Kyllä minustakin pitäisi. Muotoilin sen uudestaan.